src/lib/core/CHIPConfig.h

```diff
@@ -2108,6 +2108,27 @@
 #define CHIP_CONFIG_PERSISTED_STORAGE_KEY_TYPE const char *
 #endif
 
+/**
+ * @def CHIP_CONFIG_LIFETIIME_PERSISTED_COUNTER_KEY
+ *
+ * @brief
+ *   Lifetime Counter Key.
+ */
+#ifndef CHIP_CONFIG_LIFETIIME_PERSISTED_COUNTER_KEY
+#define CHIP_CONFIG_LIFETIIME_PERSISTED_COUNTER_KEY "rotation-counter"
+#endif
+
+/**
+ * @def CHIP_CONFIG_LIFETIIME_PERSISTED_COUNTER_INCREMENT
+ *
+ * @brief
+ *   Lifetime Counter increment.
```

Comment:
Is there a case where the increment would not be 1?

Incrementing by 1 seems to be standard enough to not require a constant. If we always increment by 1, I would drop this define.

Reply:
make sense, updated